bin/varnishd/cache/cache_vrt_vcl.c

```diff
@@ -444,6 +449,15 @@ vcl_call_method(struct worker *wrk, struct req *req, struct busyobj *bo,
 	assert(ctx.now != 0);
 	ctx.specific = specific;
 	ctx.method = method;
+	if (track_call > 0) {
```

Comment:
When can this ever be zero ?

Reply:
it gets set to zero if there are no references to subs produced
https://github.com/varnishcache/varnish-cache/pull/3163/files#diff-c974b53a6559894584abbf06417cc5e4R507